components/dropdownbutton/overview.md

```diff
@@ -85,29 +85,40 @@ The following table lists the DropDownButton parameters, except those related to
 
 The DropDownButton exposes settings for its dropdown (popup). To configure the options, declare a  `<DropDownButtonPopupSettings>` tag inside a `<DropDownButtonSettings>` tag:
 
-<div class="skip-repl"></div>
-
 ````CSHTML
-<TelerikDropDownButton>
+<TelerikDropDownButton Icon="@SvgIcon.Share">
+    <DropDownButtonContent>Share</DropDownButtonContent>
+
     <DropDownButtonSettings>
-        <DropDownButtonPopupSettings Height="150px" />
+        <DropDownButtonPopupSettings Height="auto" MaxHeight="300px" MinHeight="50px" />
     </DropDownButtonSettings>
+
+    <DropDownButtonItems>
+        <DropDownButtonItem Icon="@SvgIcon.Facebook">Facebook</DropDownButtonItem>
+        <DropDownButtonItem Icon="@SvgIcon.Twitter">Twitter</DropDownButtonItem>
+        <DropDownButtonItem Icon="@SvgIcon.Linkedin">Linkedin</DropDownButtonItem>
+        <DropDownButtonItem Icon="@SvgIcon.Reddit">Reddit</DropDownButtonItem>
+        <DropDownButtonItem Icon="@SvgIcon.Pinterest">Pinterest</DropDownButtonItem>
+    </DropDownButtonItems>
+
 </TelerikDropDownButton>
 ````
+The DropDownButton provides the following popup settings:
 
-| Parameter | Type and Default&nbsp;Value | Description |
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
```

Comment:
Why are you not using the content template, created in `features.md`?

Reply:
Here I'm not using the template, because the template itself contains information that is not applicable to the `DropDownButton`. To be more specific, the virtualization functionality is missing for the `DropDownButton` and the template contains a notes section about the virtualization, which is applicable for all other components, but not for this one.